core-bundle/src/Security/Voter/MemberGroupVoter.php

```diff
@@ -31,13 +32,26 @@ protected function supports($attribute, $subject): bool
 
     protected function voteOnAttribute($attribute, $subject, TokenInterface $token): bool
     {
+        // Filter non-numeric values
+        $subject = array_filter(array_map('intval', (array) $subject));
```

Comment:
```suggestion
        $subject = array_values(array_map('intval', array_filter((array) $subject, static function ($val) { return is_numeric($val); })));
```

Because https://3v4l.org/oNboY

Reply:
More readable solution:

```php
if (!\is_array($subject)) {
    $subject = [$subject];
}

$subject = array_values(array_filter($subject, static function ($val) { return is_numeric($val); }));

// …

if (!$user instanceof FrontendUser) {
    return \in_array(-1, $subject, false);
}
```